tests/functional/reload/runahead/flow.cylc

```diff
@@ -1,24 +1,30 @@
 [scheduler]
     UTC mode = True
     [[events]]
-        timeout = PT0.2M
-        abort on timeout = True
+        abort on inactivity = True
+        inactivity = PT10S
 [scheduling]
     initial cycle point = 2010-01-01
     final cycle point = 2010-01-05
-    runahead limit = P2 # marker
+    runahead limit = PT12H # marker
     [[graph]]
         # oops is stuck waiting task to hold back runahead
-        R1/T00 = "foo & reloader => oops"
-        T00/PT6H = "foo => bar"
+        R1 = "foo:start => reloader"
+        PT6H ="foo" 
 [runtime]
     [[foo]]
-        script = false
-    [[bar, oops]]
-        script = true
+        script = """
+            if [[ "$CYLC_TASK_JOB" == '20100101T0000Z/foo/01' ]]; then
+                # keep running until scheduler is gone
+                while cylc ping $CYLC_WORKFLOW_NAME; do
+                    sleep 10
+                done
+            else
+                true
+            fi
+        """
```

Comment:
Several functional tests updated to test the runahead limit like this, by keeping the earliest task running until the scheduler shuts down on an inactivity timeout of `PT10S`.  This is a potential cause of test flakiness under heavy load, but not sure if there's an alternative when runahead limiting is only caused by active tasks.